resources/prosody-plugins/mod_speakerstats_component.lua

```diff
@@ -0,0 +1,189 @@
+local get_room_from_jid = module:require "util".get_room_from_jid;
+local jid_resource = require "util.jid".resource;
+local ext_events = module:require "ext_events"
+local st = require "util.stanza";
+local socket = require "socket";
+local json = require "util.json";
+
+local muc_component_host = module:get_option_string("muc_component");
+if muc_component_host == nil then
+    log("error", "No muc_component specified. No muc to operate on!");
+    return;
+end
+local muc_module = module:context("conference.damencho.jitsi.net");
+if muc_module == nil then
+    log("error", "No such muc found, check muc_component config.");
+    return;
+end
+
+log("debug", "Starting speakerstats for %s", muc_component_host);
+
+-- receives messages from client currently connected to the room
+-- clients indicates their own dominant speaker events
+function on_message(event)
+    -- Check the type of the incoming stanza to avoid loops:
+    if event.stanza.attr.type == "error" then
+        return; -- We do not want to reply to these, so leave.
+    end
+
+    local speakerStats
+        = event.stanza:get_child('speakerstats', 'http://jitsi.org/jitmeet');
```

Comment:
I guess `http://jitsi.org/jitmeet` could be anything we want, but why "jitmeet" and not "jitsimeet" or something else?

Reply:
Cause we use the same namespace on several other places ...
https://github.com/jitsi/jicofo/blob/748b42f067a7495a91891969388203d94f2b44ca/src/main/java/org/jitsi/impl/protocol/xmpp/extensions/EtherpadPacketExt.java#L34
https://github.com/jitsi/jicofo/blob/a8674e80b371fe7221d80f207636ec0fa99a8723/src/main/java/org/jitsi/impl/protocol/xmpp/extensions/MuteIq.java#L35
https://github.com/jitsi/jicofo/blob/748b42f067a7495a91891969388203d94f2b44ca/src/main/java/org/jitsi/impl/protocol/xmpp/extensions/StartMutedPacketExtension.java#L34
https://github.com/jitsi/jicofo/blob/f259011ccefa901cbb588773d424b8dc68a8f9c9/src/main/java/org/jitsi/impl/protocol/xmpp/extensions/UserInfoPacketExt.java#L40